index.html

```diff
@@ -3404,6 +3397,13 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+        <p>
+The media type `application/vc+ld+json` does not imply any particular securing
+mechanism. It describes an object that is securable, but a securing mechanism
+needs to be applied to enable validation of integrity. This also applies to
```

Comment:
This would make this statement more consistent with our terminology definitions. 
```suggestion
needs to be applied to enable verification of integrity. This also applies to
```